### PR TITLE
Add Confirm all command

### PR DESCRIPTION
**Pull request by pastudan**
For when you lost a lot of matches

note: I haven't tested this at all.

### CODE REVIEW

**Comment by jbarr21** [comment]
neat!

multi-add would be nice, as well. eg `I crushed @opponent 21-10, 21-16, 21-4`

**Comment by pastudan** [comment]
@jbarr21 I'll build that when I start winning multiple matches

File: elobot.py [hunk 1]
```diff
@@ -48,7 +49,9 @@ def run(self):
                     elif WINNER_REGEX.match(message['text']):
                         self.winner(message)
                     elif CONFIRM_REGEX.match(message['text']):
-                        self.confirm(message)
+                        self.confirm(message['user'], message['text'])
+                    elif CONFIRM_ALL_REGEX.match(message['text']):
+                        _all(message)
```
Comment (mburst) [inline elobot.py]: Is this supposed to be self.confirm_all?

Comment (pastudan) [inline elobot.py]: 1b28b492cacc62ea755f4d6a8605e422cdab67e6

**Pull request merged**